tests/packages/drizzle/ws.test.ts

```diff
@@ -1,10 +1,10 @@
 import { expect, test } from 'vitest';
-import { Pool } from '../../../dist/npm';
+import { Pool } from '../../..';
```

Comment:
`@neondatabase/serverless';` just for sake of consistency

Reply:
Good spot: now changed.